gen/dcos-config.yaml

```diff
@@ -1350,6 +1350,22 @@ package:
           "registrar",
           "allocator",
        ]
+      # Reads 'mntr' stats from one or many zookeeper servers
+      [[inputs.zookeeper]]
+        ## An array of address to gather stats about. Specify an ip or hostname
+        ## with port. ie localhost:2181, 10.0.0.1:2181, etc.
+        ## If no servers are specified, then localhost is used as the host.
+        ## If no port is specified, 2181 is used
+        servers = [":2181"]
+        ## Timeout for metric collections from all servers.  Minimum timeout is "1s".
+        timeout = "10s"
+        ## Optional TLS Config
+        # enable_ssl = true
+        # tls_ca = "/etc/telegraf/ca.pem"
+        # tls_cert = "/etc/telegraf/cert.pem"
+        # tls_key = "/etc/telegraf/key.pem"
+        ## If false, skip chain & host verification
+        # insecure_skip_verify = true
```

Comment:
Is this correct? My intuitive interpretation is that setting this to `true` would cause verification to be skipped and `false` would cause verification to be done.  Either way, the default setting should be the one that cause verification to happen. All this, despite it being commented out! :smiling_imp:

Reply:
It took me some time to get what you meant here, but yes this should not be the default. I've made it explicit now to skip verification until we make ZooKeeper TLS-enabled :)

Reply:
Sorry, I think it's the multiple negatives (false, skip) that make it confusing, along with the fact that this line is inoperative until the we setup TLS.
1. In the first line, where the comment says "If false, skip...verification", I would expect the value `true` to skip verification, so I think this comment may be incorrect.
2. In the second line, to ensure we default to secure, the value should be the value that does not skip verification. If the comment is indeed correct, then `insecure_skip_verify = true` is correct. If the comment is incorrect, then this line should be `insecure_skip_verify = false`.
3. There is no need to uncomment this line until the `tls_*` lines are also uncommented.

Reply:
Thanks, I've looked up the code and updated the config description accordingly.